aip/aog/3010.md

```diff
@@ -0,0 +1,55 @@
+---
+aip:
+  id: 3010
+  state: draft
+  created: 2020-01-14
+  updated: 2020-01-14
+  type: guidance
+  scope: aog
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /aog/3010
+---
+
+# Actions on Google Vertical Integration Webhook Format
+
+If an action is part of a vertical program that uses Google Fulfillment, it
+usually needs to expose a webhook that Google will call to execute user queries
+via a structured protocol. This AIP defines requirement for the webhook format.
+
+## Guidance
+
+### Base URL in Action Package
+
+The `baseUrl` configured in Action Package defines the common part of the
```

Comment:
Can we link "Action Package" to public docs?

Reply:
The action package referenced in public docs doesn't have GF or the field that's mentioned here...

Reply:
Should we remove the reference to the `baseUrl` field, if it's not in public docs (and if we're taking the same approach with mentions of GF)?

Reply:
alright, I just say it's base URL in action package.